vendor/k8s.io/kubernetes/pkg/kubectl/cmd/create_service.go

```diff
@@ -133,6 +133,7 @@ func NewCmdCreateServiceNodePort(f *cmdutil.Factory, cmdOut io.Writer) *cobra.Co
 	cmdutil.AddValidateFlags(cmd)
 	cmdutil.AddPrinterFlags(cmd)
 	cmdutil.AddGeneratorFlags(cmd, cmdutil.ServiceNodePortGeneratorV1Name)
+	cmd.Flags().Int32("nodeport", 0, "Port used to expose the service on each node in a cluster.")
```

Comment:
Seems like it should be named "node-port" based on the capitalization used in the API (NodePort)